src/styles/blocks/header/_navigation.scss

```diff
@@ -0,0 +1,35 @@
+.nav__list {
```

Comment:
Where is nav block? I see only elements, but they can't exist without a block, according to BEM